### PR TITLE
docs | concepts | workloads | controllers | statefulset

### PR DESCRIPTION
**Pull request by feloy**

### CODE REVIEW

File: content/fr/docs/concepts/workloads/controllers/statefulset.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
+---
+reviewers:
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
```
Comment (sftim) [inline content/fr/docs/concepts/workloads/controllers/statefulset.md:8]: BTW, remember to snip out the list of reviewers; these are English speakers (I assume).

**Comment by kbhawkey** [comment]
Hello @feloy .
Are you actively working on this PR?
/cc @kubernetes/sig-docs-fr-reviews

**Comment by feloy** [comment]
> Hello @feloy .
> Are you actively working on this PR?
> /cc @kubernetes/sig-docs-fr-reviews

Not very actively I have to confess, but yes, I would like to finish the translation or this page

**Comment by sftim** [comment]
I might be able to help with suggestions, then you can correct any slips.

**Review by Aut0R3V — APPROVED** [review]

**Comment by Aut0R3V** [comment]
My approval was for the document in English.

**Comment by sftim** [comment]
@feloy if you're still working on this, I recommend you rebase it so that the preview uses the latest submodules and latest configured Hugo version.

File: content/fr/docs/concepts/workloads/controllers/statefulset.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+Chaque Pod dans un StatefulSet dérive son nom d'hôte du nom du StatefulSet
+et de l'ordinal du Pod. Le modèle pour le nom d'hôte généré est
+`$(nom statefulset)-$(ordinal)`. L'exemple ci-dessus créera trois Pods
+nommés `web-0,web-1,web-2`.
+Un StatefulSet peut utiliser un [Service Headless](/docs/concepts/services-networking/service/#headless-services)
+pour contrôler le domaine de ses Pods. Le domaine pris en charge par ce Service prend la forme :
+`$(nom service).$(namespace).svc.cluster.local`, où "cluster.local" est le domaine du cluster.
+Chaque fois qu'un Pod est créé, il obtient un sous-domaine DNS correspondant, prenant la forme :
+`$(nom pod).$(domaine du service gouvernant)`, où le service gouvernant est défini par par le
```
Comment (remyleone) [inline content/fr/docs/concepts/workloads/controllers/statefulset.md:132]:
```suggestion
`$(nom pod).$(domaine du service gouvernant)`, où le service gouvernant est défini par le
```

File: content/fr/docs/concepts/workloads/controllers/statefulset.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+Le domaine cluster sera `cluster.local` à moins qu'il soit
+[configuré autrement](/docs/concepts/services-networking/dns-pod-service/).
+{{< /note >}}
+
+### Stockage stable
+
+Kubernetes crée un [PersistentVolume](/docs/concepts/storage/persistent-volumes/) pour chaque
+VolumeClaimTemplate. Dans l'exemple nginx ci-dessus, chaque Pod se verra affecter un unique PersistentVolume
+avec un StorageClass de `my-storage-class` et 1 Gib de stockage provisionné. Si aucun StorageClass
+n'est spécifié, alors le StorageClass par défaut sera utilié. Lorsqu'un Pod est (re)schedulé
```
Comment (remyleone) [inline content/fr/docs/concepts/workloads/controllers/statefulset.md:169]:
```suggestion
n'est spécifié, alors le StorageClass par défaut sera utilisé. Lorsqu'un Pod est (re)schedulé
```

**Comment by feloy** [comment]
Thanks @remyleone for the review

**Pull request merged**